enterprise/static/enterprise/enterprise_course_enrollment_page.js

```diff
@@ -0,0 +1,60 @@
+function focusableElementsSelector() {
+    return [
+        'a[href], area[href], input:not([disabled]), select:not([disabled]),',
+        'textarea:not([disabled]), button:not([disabled]), iframe, object, embed,',
+        '*[tabindex], *[contenteditable]'
+    ].join(' ');
+}
+function enableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+        focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'false');
+    focusableItems.attr('tabindex', '0');
+}
+function disableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+    focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'true');
+    focusableItems.attr('tabindex', '-1');
+}
+function showModal(modalSelector) {
+    $(modalSelector).attr('aria-hidden', 'false');
+    $(modalSelector).show();
+    $("#modal-close-button").focus();
+    $('body').addClass('open-modal');
+    disableTabIndexingOn('.enterprise-container');
+}
+function hideModal(modalSelector) {
+    $(modalSelector).attr('aria-hidden', 'true');
+    $(modalSelector).hide();
+    $('body').removeClass('open-modal');
+    enableTabIndexingOn('.enterprise-container');
+    $("#view-course-details-link").focus();
+}
```

Comment:
@replaceafill Sorry, I should have provided more details about the approach I had in mind. Using [`toggle`](https://api.jquery.com/toggle/#toggle-display) and [`toggleClass`](https://api.jquery.com/toggleClass/), you can save more than one line:

```js
function toggleModal(modalSelector, visible) {
    var ariaHidden = 'true',
        tabIndex = '-1',
        focusSelector = '#view-course-details-link';

    if (visible) {
        ariaHidden = 'false';
        tabIndex = '0';
        focusSelector = '#modal-close-button';
    }

    $(modalSelector).toggle(visible);
    $('body').toggleClass('open-modal', visible);

    $(modalSelector).attr('aria-hidden', ariaHidden);
    $(focusSelector).focus();
    setTabIndexing('.enterprise-container', ariaHidden, tabIndex);
}
```

I would keep the conditional here (instead of introducing an `ariaHidden` parameter) to make it clear that the value of the `aria-hidden` attribute depends on the target state of the modal, and to keep control over setting that attribute inside the `toggleModal` function. When calling `toggleModal`, make sure to pass a Boolean value for `visible` (`toggleClass` requires this).

Note also that with this approach, `enableTabIndexingOn` and `disableTabIndexingOn` are no longer needed.